src/ui/popup/edit.tsx

```diff
@@ -133,7 +133,19 @@ export default function Edit(props: { tab: Resource<ManagerTab> }) {
 							/>
 							<Squircle id="coverArtClip" />
 						</a>
-						<div class={styles.songDetails}>
+						<div
+							class={styles.songDetails}
+							onKeyDown={(event) => {
+								if (event.key === 'Enter') {
```

Comment:
```suggestion
								if (event.key === 'Enter' && !event.isComposing) {
```

Reply:
This causes errors with some IMEs for languages like chinese and japanese

Reply:
if statement updated at https://github.com/web-scrobbler/web-scrobbler/pull/3785/commits/d5d993d8a34833f897bbcf2a61196ee5cd363ab1